### PR TITLE
python3Packages.smbprotocol: 1.5.1 -> 1.6.1

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
Update to latest upstream release 1.6.1

Change log:
- https://github.com/jborean93/smbprotocol/releases/tag/v1.6.1
- https://github.com/jborean93/smbprotocol/releases/tag/v1.6.0

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Relase notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 131880` at 90b8e6fa run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>20 packages failed to build:</summary>
  <ul>
    <li><a href="https://gist.github.com/5eb57faed5c4dbfb9fa116439dca6886">python38Packages.aplpy</a></li>
    <li><a href="https://gist.github.com/575c37c48e9de651cad43224e9bbb423">python38Packages.asdf</a></li>
    <li><a href="https://gist.github.com/3486b178d4369500cc40a83956b9dbc5">python38Packages.dask-jobqueue</a></li>
    <li><a href="https://gist.github.com/647b095fcab0c6c8b464d80e04b98daf">python38Packages.dask-xgboost</a></li>
    <li><a href="https://gist.github.com/de8beac78a13ac986651a741b4c29e72">python38Packages.pyregion</a></li>
    <li><a href="https://gist.github.com/12632406e82daff94c089b97c6516b07">python38Packages.pytest-astropy</a></li>
    <li><a href="https://gist.github.com/17d3c7024bd878f6be43d5c9f16e229e">python38Packages.reproject</a></li>
    <li><a href="https://gist.github.com/b5355dc5544338b67757242810cf5a15">python38Packages.sunpy</a></li>
    <li><a href="https://gist.github.com/eeba49c212d816a521271b7207a221d2">python38Packages.tokenizers</a></li>
    <li><a href="https://gist.github.com/e4ed56eeba7500fa047fc0d7649eb37f">python38Packages.transformers</a></li>
    <li><a href="https://gist.github.com/72ff050c8c4f75a629eeba8e99d2457f">python39Packages.aplpy</a></li>
    <li><a href="https://gist.github.com/6b8c9bd304bd85db31ef02e20e6205c2">python39Packages.asdf</a></li>
    <li><a href="https://gist.github.com/3690e75226504e1dde88283c8c4aa9b5">python39Packages.dask-jobqueue</a></li>
    <li><a href="https://gist.github.com/ec5247d04b5d6c79b858f30b17807084">python39Packages.dask-xgboost</a></li>
    <li><a href="https://gist.github.com/b13b2791c5eddb6416f4368d7dafbfea">python39Packages.pyregion</a></li>
    <li><a href="https://gist.github.com/88cfb38bccdb18f15d8642dce8e24437">python39Packages.pytest-astropy</a></li>
    <li><a href="https://gist.github.com/bebc8a0bc3fc08b6adfc867c2b68d7a3">python39Packages.reproject</a></li>
    <li><a href="https://gist.github.com/6b28a74d5e4c7cb4e524ea194d8b7c45">python39Packages.sunpy</a></li>
    <li><a href="https://gist.github.com/98809c95c73b2966404867956ae51262">python39Packages.tokenizers</a></li>
    <li><a href="https://gist.github.com/bd90a3c86b2385e7401f13fd136f6924">python39Packages.transformers</a></li>
  </ul>
</details>
<details>
  <summary>24 packages skipped due to time constraints:</summary>
  <ul>
    <li>python38Packages.caffe</li>
    <li>python38Packages.clifford</li>
    <li>python38Packages.dask-glm</li>
    <li>python38Packages.dask-ml</li>
    <li>python38Packages.datashader</li>
    <li>python38Packages.imgaug</li>
    <li>python38Packages.mask-rcnn</li>
    <li>python38Packages.rl-coach</li>
    <li>python38Packages.spacy-transformers</li>
    <li>python38Packages.spectral-cube</li>
    <li>...</li>
  </ul>
</details>
<details>
  <summary>48 packages built successfully:</summary>
  <ul>
    <li>python38Packages.batchgenerators</li>
    <li>python38Packages.dask</li>
    <li>python38Packages.dask-gateway</li>
    <li>python38Packages.dask-image</li>
    <li>python38Packages.dask-mpi</li>
    <li>python38Packages.datasets</li>
    <li>python38Packages.distributed</li>
    <li>python38Packages.fastparquet</li>
    <li>python38Packages.fsspec</li>
    <li>python38Packages.gcsfs</li>
    <li>python38Packages.glymur</li>
    <li>python38Packages.image-match</li>
    <li>python38Packages.imagecorruptions</li>
    <li>python38Packages.intake</li>
    <li>python38Packages.pims</li>
    <li>python38Packages.pyfftw</li>
    <li>python38Packages.pytest-astropy-header</li>
    <li>python38Packages.s3fs</li>
    <li>python38Packages.scikitimage</li>
    <li>python38Packages.slicedimage</li>
    <li>python38Packages.smbprotocol</li>
    <li>python38Packages.sparse</li>
    <li>python38Packages.stytra</li>
    <li>python38Packages.tensorly</li>
    <li>python39Packages.batchgenerators</li>
    <li>python39Packages.dask</li>
    <li>python39Packages.dask-gateway</li>
    <li>python39Packages.dask-image</li>
    <li>python39Packages.dask-mpi</li>
    <li>python39Packages.datasets</li>
    <li>python39Packages.distributed</li>
    <li>python39Packages.fastparquet</li>
    <li>python39Packages.fsspec</li>
    <li>python39Packages.gcsfs</li>
    <li>python39Packages.glymur</li>
    <li>python39Packages.image-match</li>
    <li>python39Packages.imagecorruptions</li>
    <li>python39Packages.intake</li>
    <li>python39Packages.pims</li>
    <li>python39Packages.pyfftw</li>
    <li>python39Packages.pytest-astropy-header</li>
    <li>python39Packages.s3fs</li>
    <li>python39Packages.scikitimage</li>
    <li>python39Packages.slicedimage</li>
    <li>python39Packages.smbprotocol</li>
    <li>python39Packages.sparse</li>
    <li>python39Packages.stytra</li>
    <li>python39Packages.tensorly</li>
  </ul>
</details>

Note that build failures [may predate](https://github.com/nix-community/hydra-check) this PR, and could be nondeterministic or hardware dependent.
Please exercise your independent judgement. Does something look off? Please file an [issue](https://github.com/rmcgibbo/r-rmcgibbo/issues) or reach out on IRC.

**Comment by siraben** [comment]
I think `python38Packages.imgaug` has an infinite looping test as a result of this PR (when I ran `nixpkgs-review` it didn't terminate).

**Comment by fabaff** [comment]
`python3Packages.imgaug` build locally for me.

**Comment by SuperSandro2000** [comment]
This is a semi-automatic executed nixpkgs-review with [nixpkgs-review-checks extension](https://github.com/SuperSandro2000/nixpkgs-review-checks). It is checked by a human on a best effort basis and does not build all packages (e.g. lumo, tensorflow or pytorch).

Result of `nixpkgs-review pr 131880` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
<summary>
8 packages marked as broken and skipped:
</summary>
<ul>
<li>python38Packages.clifford</li>
<li>python38Packages.dask-jobqueue</li>
<li>python38Packages.dask-xgboost</li>
<li>python38Packages.rl-coach</li>
<li>python39Packages.clifford</li>
<li>python39Packages.dask-jobqueue</li>
<li>python39Packages.dask-xgboost</li>
<li>python39Packages.rl-coach</li>
</ul>
</details>
<details>

  <summary>22 packages failed to build and already failed to build on hydra master:</summary>
  <ul>
<li>python38Packages.aplpy: log was empty</li>
<li>python38Packages.asdf: log was empty</li>
<li>python38Packages.gcsfs: log was empty</li>
<li>python38Packages.pyregion: log was empty</li>
<li>python38Packages.pytest-astropy: log was empty</li>
<li>python38Packages.reproject: log was empty</li>
<li>python38Packages.spacy-transformers: log was empty</li>
<li>python38Packages.spectral-cube: log was empty</li>
<li>python38Packages.sunpy: log was empty</li>
<li>python38Packages.tokenizers: log was empty</li>
<li>python38Packages.transformers: log was empty</li>
<li>python39Packages.aplpy: log was empty</li>
<li>python39Packages.asdf: log was empty</li>
<li>python39Packages.gcsfs: log was empty</li>
<li>python39Packages.pyregion: log was empty</li>
<li>python39Packages.pytest-astropy: log was empty</li>
<li>python39Packages.reproject: log was empty</li>
<li>python39Packages.spacy-transformers: log was empty</li>
<li>python39Packages.spectral-cube: log was empty</li>
<li>python39Packages.sunpy: log was empty</li>
<li>python39Packages.tokenizers: log was empty</li>
<li>python39Packages.transformers: log was empty</li>
  </ul>
</details>
<details>

  <summary>1 package failed to build and are new build failure:</summary>
  <ul>
<li>python38Packages.stumpy: log was empty</li>
  </ul>
</details>
<details>
<summary>
61 packages built:
</summary>
<ul>
<li>python38Packages.batchgenerators</li>
<li>python38Packages.caffe</li>
<li>python38Packages.dask</li>
<li>python38Packages.dask-gateway</li>
<li>python38Packages.dask-glm</li>
<li>python38Packages.dask-image</li>
<li>python38Packages.dask-ml</li>
<li>python38Packages.dask-mpi</li>
<li>python38Packages.datasets</li>
<li>python38Packages.datashader</li>
<li>python38Packages.distributed</li>
<li>python38Packages.fastparquet</li>
<li>python38Packages.fsspec</li>
<li>python38Packages.glymur</li>
<li>python38Packages.image-match</li>
<li>python38Packages.imagecorruptions</li>
<li>python38Packages.imgaug</li>
<li>python38Packages.intake</li>
<li>python38Packages.mask-rcnn</li>
<li>python38Packages.pims</li>
<li>python38Packages.pyfftw</li>
<li>python38Packages.pytest-astropy-header</li>
<li>python38Packages.s3fs</li>
<li>python38Packages.scikitimage</li>
<li>python38Packages.slicedimage</li>
<li>python38Packages.smbprotocol</li>
<li>python38Packages.sparse</li>
<li>python38Packages.streamz</li>
<li>python38Packages.stytra</li>
<li>python38Packages.tensorly</li>
<li>python39Packages.batchgenerators</li>
<li>python39Packages.caffe</li>
<li>python39Packages.dask</li>
<li>python39Packages.dask-gateway</li>
<li>python39Packages.dask-glm</li>
<li>python39Packages.dask-image</li>
<li>python39Packages.dask-ml</li>
<li>python39Packages.dask-mpi</li>
<li>python39Packages.datasets</li>
<li>python39Packages.datashader</li>
<li>python39Packages.distributed</li>
<li>python39Packages.fastparquet</li>
<li>python39Packages.fsspec</li>
<li>python39Packages.glymur</li>
<li>python39Packages.image-match</li>
<li>python39Packages.imagecorruptions</li>
<li>python39Packages.imgaug</li>
<li>python39Packages.intake</li>
<li>python39Packages.mask-rcnn</li>
<li>python39Packages.pims</li>
<li>python39Packages.pyfftw</li>
<li>python39Packages.pytest-astropy-header</li>
<li>python39Packages.s3fs</li>
<li>python39Packages.scikitimage</li>
<li>python39Packages.slicedimage</li>
<li>python39Packages.smbprotocol</li>
<li>python39Packages.sparse</li>
<li>python39Packages.streamz</li>
<li>python39Packages.stumpy</li>
<li>python39Packages.stytra</li>
<li>python39Packages.tensorly</li>
</ul>
</details>
